lib/node_modules/@stdlib/math/base/special/bernoullif/manifest.json

```diff
@@ -37,11 +37,11 @@
       "libpath": [],
       "dependencies": [
         "@stdlib/math/base/napi/unary",
-        "stdlib/math/base/assert/is_nonnegative_integer.h",
-        "stdlib/math/base/assert/is_nanf.h",
-        "stdlib/math/base/assert/is_odd.h",
-        "stdlib/constants/float32/ninf.h",
-        "stdlib/constants/float32/pinf.h"
+        "@stdlib/math/base/assert/is_nonnegative_integer.h",
```

Comment:
These deps are still incorrect. You need to list package names, not paths to header files.